game/weather.rpy

```diff
@@ -0,0 +1,814 @@
+# Weather data
+default persistent.weather_api_key = None
+default persistent.weather_validate_apikey_in_time = None
+default persistent.is_weather_tracking_set_up = False
+default persistent.current_weather_type = weather.TYPE_CLEAR
+default persistent.current_weather_long = dict()
+
+# Location data
+default persistent.latitude = None
+default persistent.longitude = None
+default persistent.jn_hemisphere_north_south = None
+default persistent.jn_hemisphere_east_west = None
+
+init -1 python in weather:
+    import urllib2
+    import datetime
+    import store
+
+    #TEST API key, feel free to use
+    #2c2f369ad4987a01f5de4c149665c5fd
+    #DEBUG: TODO: remove in production
```

Comment:
Could we perhaps move this to the `dev` folder under `game`? That way we can better make sure it isn't missed when we transition to production/release, as it'll be more visible as a separate document entirely than tucked away in source code.